frontend/src/pages/eventoverview.js

```diff
@@ -0,0 +1,216 @@
+              <ModalBody>
+                {alert}
+                <Form>
+                  <FormGroup>
+                    <Label>Name</Label>
+                    <Input
+                      name="name"
+                      placeholder="i.e. Product Showcase"
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Date</Label>
+                    <Input
+                      name="date"
+                      type="date"
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Start Time</Label>
+                    <Input
+                      name="startTime"
+                      type="time"
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>End Time</Label>
+                    <Input
+                      name="endTime"
+                      type="time"
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Location</Label>
+                    <Input
+                      name="location"
+                      placeholder="i.e. ECEB"
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Description</Label>
+                    <Input
+                      name="description"
+                      type="textarea"
+                      placeholder="i.e. Teams present their final product."
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Facebook Event Link</Label>
+                    <Input
+                      name="fbLink"
+                      type="url"
+                      placeholder="i.e. https://www.facebook.com/events/2405129342892922/"
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                </Form>
+              </ModalBody>
+              <ModalFooter>
+                <Button color="secondary" onClick={this.toggleModal}>
+                  Cancel
+                </Button>
+                <Button color="primary" onClick={this.addEvent}>
```

Comment:
`<Button type="submit color="primary" onClick={this.addEvent}>`

Reply:
rip it doesn't work

Reply:
That's because buttons don't have a `type` prop: https://reactstrap.github.io/components/buttons/